test.e2e/spec/services.spec.js

```diff
@@ -181,6 +181,89 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
             }
           );
         });
+
+        it('intercepts internal requests with internal urlBase',function() {
+          var urlBase = '/api',
+              nonUrlBase = '/otherlocalpath',
+              authHeader = 'X-Test-Token',
+              accessTokenId = '12345';
```

Comment:
`authHeader` and `accessTokenId` are duplicated in several tests. Could you please extract them to shared constants, e.g. `AUTH_HEADER` and `ACCESS_TOKEN_ID`